Project-Descriptions-and-Plans/CORE-V-TRUSTED-MCU/Preliminary-project-proposal-for-Trusted-MCU-SoC.md

```diff
@@ -0,0 +1,248 @@
+
+## Components
+The project comprises two main components.
+
+### Component 1 - Design and Fabrication of the SoC Device
+* Design [effort | timeline]
+    - Interconnect update to include address filtering
+    - Power Management: Define and Partition the SoC into power domains
+    - Enable Priority based arbitration mechanism on the TCM memory interface
+* Integration [effort | timeline]
+    - Repository for the Trusted MCU RTL
+    - Event unit update
+    - FPGA Implementation
+        - Synthesis and bitstream generation
+        - Test firmware
+    - ASIC Implementation
+        - Synthesis
+        - Power Management - UPF for Power Gating
+        - RTL to GDS2 conversion
+* UVM Verification [effort | timeline]
+    - uDMA verification and associated peripherals
+        - Verification Plan
+        - Register Model
+        - UVM test benches and environment
+    - APB peripheral verification (Timer, Advanced Timer, I2C, UART, etc.)
+        - Verification Plan
+        - Register Model
+        - UVM test benches and environment
+    - SoC top level verification
+        - Replace the two cores with OBI agents
+        - Test hardware reset and register bash for all IPs
+* Firmware [effort | timeline]
+    - Secure boot firmware 
+    - Symmetric key protection
+    - Asymmetric key protection
+    - Interprocessor communication
+
+### Component 2 - Design and Build of the Evaluation Kit
+[Detailed timeline to be provided]
+* Board based on the MCU
+* SDK based on the SDK of the MCU
+* Documentation: lightweight user guide of the device such that a competent hardware or software engineer can evaluate the device on the dev kit.
+
+### Summary of Timeline
+
+* Architecture			Q4 2022
+* Specification			Q1 2023
+* RTL Integration
+* FPGA Prototyping		Q1 2023
+* UVM Verification
+* RTL to GDS			Q2 2023
+* Development kit
+* Firmware
+* Tapeout				Q3 2023
+
+
+## OpenHW Members/Participants committed to participate in CORE-V MCU project
+
+* Low Power Futures
+* Datum TC
+* Quick Logic
+* Crypto Quantique
+
+## Technical Project Leader(s) (TPLs)
+
+Abdoulaye Berthe
+
+## Project Manager (PM)
+Duncan Bee
+
+## Project Documents
+The following project documents will be created:
+
+* Lightweight specification for the test chip
+* Lightweight user guide of the device such that a competent hardware or software engineer can write software to evaluate/measure the device capabilities.
+* Lightweight datasheets include operating conditions, AC/DC parameters, basic timing information, booting instructions, pin muxing tables, register definitions, etc.
+* Ideally, this documentation would be auto-created using a Sphinx-like tool.
+* Reference documentation for all UVM IP including user guides
+
+## Summary of requirements
+Create an SoC that is suitable for evaluation:
+
+* Use a big/little configuration CV32E40S/CV32E20
+* Select and prototype on an FPGA - Nexys or Genesys
+* Same peripheral set as CV32E40S
```

Comment:
The CV32E40S does not have explicit support for any peripherals.  Did you mean `same peripheral set as CORE-V-MCU`?